.github/workflows/ci.yml

```diff
@@ -70,7 +70,7 @@ jobs:
         uses: actions/download-artifact@v3
         with:
           name: npm-dist
-          path: dist
+          path: packages/legacy/dist
       - name: Publish 🚀
         uses: JS-DevTools/npm-publish@541aa6b21b4a1e9990c95a92c21adc16b35e9551
         with:
```

Comment:
We'll need to add the [`package` input](https://github.com/JS-DevTools/npm-publish#usage-1) to `with` to make sure we publish from the right place:

```yaml
with:
  token: ${{ secrets.NPM_TOKEN }}
  package: packages/legacy
```

While testing, you can also try things out with the `dry-run` input to validate the correct files are being packed and published